engine/evalOp.cc

```diff
@@ -767,15 +767,21 @@ namespace minsky
                 e.shape=from1.dims();
                 if (to.xVector!=from1.xVector)
                   to.setXVector(from1.xVector);
-                for (size_t i=0; i<from1.numElements(); ++i)
-                  t->in1.push_back(i+from1.idx());
+                  
+                // For feature 47
+                  for (size_t i=0; i<from1.dataSize(); ++i)                          
+                    t->in1.push_back(i+from1.idx());
```

Comment:
For these operations, we probably need to transfer to index vector to to as well.

Reply:
I still need to do this.